blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java

```diff
@@ -562,6 +562,26 @@ public boolean notifyCharacteristicChanged(@NotNull final byte[] value, @NotNull
         return result;
     }
 
+    /**
+     * Send a notification or indication that a local characteristic has been
+     * updated to the central passed in
+     *
+     * <p>A notification or indication is sent to the remote centrals to signal
+     * that the characteristic has been updated. If the connected central is not
+     * found then false is return
+     *
+     * @param characteristic the characteristic for which to send a notification
+     * @return true if the central exists and the operation was enqueued, otherwise false
+     */
+    public boolean notifyCharacteristicChanged(@NotNull final byte[] value, @NotNull final BluetoothCentral bluetoothCentral, @NotNull final BluetoothGattCharacteristic characteristic){
+        BluetoothDevice bluetoothDevice = getDeviceForCentral(bluetoothCentral);
```

Comment:
Please add checks to confirm that parameters are not null:

        Objects.requireNonNull(bluetoothCentral, CENTRAL_IS_NULL);
        Objects.requireNonNull(value, CHARACTERISTIC_VALUE_IS_NULL);
        Objects.requireNonNull(characteristic, CHARACTERISTIC_IS_NULL);

Reply:
Added to top of method